.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Avocado Gating CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Logical, stylistic, analytical and formatter checks
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
```

Comment:
Where does this line come from?

Reply:
This is using public actions that are very well recommended and maintained. Like Ansible playbooks.

Reply:
Okay, I'll take a look.